libsolidity/parsing/Parser.cpp

```diff
@@ -65,6 +65,8 @@ class Parser::ASTNodeFactory
 		return make_shared<NodeType>(m_location, std::forward<Args>(_args)...);
 	}
 
+	SourceLocation const& location() const noexcept { return m_location; }
```

Comment:
Oh, did we agree on using `noexcept` where applicable?

Reply:
I agreed with myself to be in line with the CppCoreGuidelines. Please Overwrite my configuration in case of violation 🔢

Reply:
Don't get me wrong. I agree with this modernization :)